README.md

```diff
@@ -22,6 +22,8 @@ $ cd mobile-apps-article-templates
 $ npm install
 ```
 
+* If you get an error on this step regarding `lib/node_modules/npm/node_modules/promise-retry/index.js` or another module, running `npm install -g npm` and re-running `npm install` may fix it.
```

Comment:
Really? Do you happen to know why?

Reply:
Not other than "it fixed it". 😬   I noticed further up the current `Readme` it also mentions `You may need to install npm globally npm install -g npm`, this is the specific case where it fixed thins for me.

Reply:
I'm a bit puzzled by that since `nvm` is meant to handle these kind of issues, but 🤷